client/my-sites/importer/importer-action-buttons/done-button.jsx

```diff
@@ -32,23 +34,38 @@ export class DoneButton extends React.PureComponent {
 
 	handleClick = () => {
```

Comment:
What do you think to relabelling the button to something more suggestive like 'View Site' or something similar?

Reply:
I'd be onboard with changing the button text, but let's push it to a follow up.